mne_nirs/io/snirf/_snirf.py

```diff
@@ -213,8 +239,24 @@ def _store_probe_landmarks(raw, probe):
         if ident is not None:
             digname.append(ident[1])
         else:
-            digname.append(str(dig.get('ident')))
+            digname.append(f"HP_{str(dig.get('ident'))}")
         diglocs[idx, :] = dig.get('r')
+
+    if atlasviewer:
+        # First, get the template positions in mni fsaverage space
+        montage = make_standard_montage(
+            'standard_1020', head_size=0.09700884729534559)
+        ch_names = montage.ch_names
+        montage_locs = np.array(list(montage._get_ch_pos().values()))
+
+        # montage_locs = np.array([d['r'] for d in montage.dig])
+        head_mri_t, _ = _get_trans('fsaverage', 'mri', 'head')
+        locations_head = apply_trans(head_mri_t, montage_locs)
+
+        for template_idx in range(len(ch_names)):
+            digname.append(ch_names[template_idx])
+            diglocs = np.vstack((diglocs, locations_head[template_idx, :]))
```

Comment:
Simpler and vectorized?
```suggestion
        digname.extend(ch_names)
        diglocs = np.concatenate((diglocs, locations_head))
```